File: Binary matrix having maximum number of 1s.py
class Solution:
    def findMaxRow(self, mat, N):
        # Code here
        j = n-1
        ind = 0
        for i in range(N):
            while j >= 0 and mat[i][j] == 1:
                j-=1
                ind = i
        return [ind, n - j - 1]

File: Binary matrix having maximum number of 1s.cpp
class Solution {
public:
    vector<int> findMaxRow(vector<vector<int>> mat, int N) {
        int row = 0, i = 0, j = N - 1;
        for (; i < N; i++) {
            while (mat[i][j] == 1 && j >= 0) {
                row = i;
                j--;
            }
        }
        return {row, N - 1 - j}; 
    }
};
